application/tests/unit/common/models/ResetTest.php

```diff
@@ -234,7 +234,29 @@ public function testSendMethodEmail()
         $this->assertTrue(EmailUtils::hasEmailFileBeenCreated($this->tester, 'email-1456769679@domain.org'));
         $this->assertTrue(EmailUtils::hasEmailFileBeenCreated($this->tester, 'requested a password change for their'));
         $this->assertEquals($attempts + 1, $reset->attempts);
+    }
+
+    public function testSendDoNotDisclose()
+    {
+        $this->markTestSkipped('test is broken because fake methods are not accessible in this context');
```

Comment:
I'm not sure I understand... what was blocking this test from working?

Reply:
There is a `_before` method that inserts a bunch of fake data into the broker test container. Problem is, there's no easy way to verify the inserted methods. Without that, this test won't send an email to the fake method. I either need to write some code that will receive the verification code and submit that for verification, or I need to use a mocked broker instead of a container. I intend to do the latter.